.github/workflows/build-assets.yml

```diff
@@ -484,3 +516,121 @@ jobs:
+          mv htmlhelp.chm ECLReference.chm
+          cp ECLReference.chm ../ECLIDE/docs/
+
+      - name: "Create PT_BR CHM file"
+        shell: "bash"
+        run: |
+          dep_tag="${{ needs.preamble.outputs.community_tag }}"
+          hpcc_version="${{ needs.preamble.outputs.hpcc_version }}"
+          curl -sL -o html_help_PT_BR.zip https://github.com/hpcc-systems/HPCC-Platform/releases/download/${dep_tag}/html_help_PT_BR-${hpcc_version}.zip
+          7z x html_help_PT_BR.zip
+          mv html_help html_help_PT_BR
+          cd html_help_PT_BR
+          # Chocolate install a package but it is not in PATH variable. Use path full of hhc.exe
+          # hhc.exe return non-zero. Appending "true" to work-around
+          C:/"Program Files (x86)"/"HTML Help Workshop"/hhc.exe htmlhelp.hhp || true
+          mv htmlhelp.chm ECLReference_PT_BR.chm
+          cp ECLReference_PT_BR.chm ../ECLIDE/docs/
+
+      - name: "Fetch Clienttools package"
+        shell: "bash"
+        run: |
+          cd ./ECLIDE
+          mkdir HPCC-Platform
+          cd HPCC-Platform
+          dep_tag="${{ needs.preamble.outputs.community_tag }}"
+          curl -sL -o hpccsystems-clienttools-${dep_tag}Windows-x86_64.exe https://github.com/hpcc-systems/HPCC-Platform/releases/download/${dep_tag}/hpccsystems-clienttools-${dep_tag}Windows-x86_64.exe
+
+      - name: "Create code sining"
+        shell: "pwsh"
+        env:
+          SIGN_CERT: ${{ secrets.SIGN_CERT }}
+          SIGN_PW: ${{ secrets.SIGN_PW }}
+        run: |
+          mkdir -p ./sign
+          cd sign
+          $pfxPath = "hpcc_code_signing.pfx"
+          $encodedBytes = [System.Convert]::FromBase64String($env:SIGN_CERT)
+          $currentDirectory = Get-Location
+          $certificatePath = Join-Path -Path $currentDirectory -ChildPath $pfxPath
+          [IO.File]::WriteAllBytes("$certificatePath", $encodedBytes)
+          echo ${env:SIGN_PW} > passphrase.txt
+          CertUtil -p ${env:SIGN_PW} hpcc_code_signing.pfx
+
+      - name: "Build ECLIDE"
+        run: |
+          cd ECLIDE
+          mkdir -p ./build
+          cd ./build
+          cmake -A Win32 -S ../ -B .
+          cmake --build . --config RelWithDebInfo --target package --parallel
+
+      - name: Sign ECLIDE Package
+        shell: "pwsh"
+        env:
+          SIGN_PW: ${{ secrets.SIGN_PW }}
+        run: |
+          cd ./ECLIDE/build
+          C:\"Program Files (x86)"\"Microsoft SDKs"\ClickOnce\SignTool\signtool.exe sign /debug /f ../../sign/hpcc_code_signing.pfx /p $env:SIGN_PW /t http://timestamp.digicert.com /fd SHA256 hpccsystems-eclide*.exe
+
+      - name: Upload Assets
+        uses: ncipollo/release-action@v1.12.0
+        with:
+          allowUpdates: true
+          generateReleaseNotes: false
+          prerelease: ${{ contains(github.ref, '-rc') }}
+          artifacts: "./ECLIDE/build/*.exe,./build/*.exe,./build/*.msi,./build/*.dmg,./build/*.pkg,./build/*.tar.gz"
```

Comment:
Just `./ECLIDE/build/*.exe` should be needed?

Reply:
Fixed